persistence/in-memory_gatewaydedupe_[,7.1.10).partial.md

```diff
@@ -0,0 +1,4 @@
+## Gateway deduplication
+
+WARNING: The in memory gateway deduplication persistence contains a memory leak. It is advised to upgrade.
```

Comment:
We generally don't document bugs in such a way. I'd either say nothing or something like "gateway deduplication has no built-in cleanup". Here's how you should clean up yourself. And add a snippet to resolve the gateway deduplication thing and call that public method that was there to erase the entries.

Reply:
@SzymonPobiega The method is public but the type is not. I'm not really comfortable with documenting such a workaround that would require reflection. I do think its strange what we wouldn't document know pitfalls.
```suggestion
In memory gateway deduplication persistence has no built-in cleanup.
```